testing/middleware/builder/builder.go

```diff
@@ -690,6 +838,36 @@ func (p *Builder) retrievePendingBlockDeneb() (*v1.ExecutionPayloadDenebWithValu
 	return denebPayload, nil
 }
 
+//func (p *Builder) retrievePendingBlockElectra() (*v1.ExecutionPayloadElectraWithValueAndBlobsBundle, error) {
```

Comment:
I think this doesn't have to be commented and you can resolve the TODO inside this function in this PR.